notebook/services/contents/tests/test_contents_api.py

```diff
@@ -70,7 +71,7 @@ def read(self, path, type=None, format=None, content=None):
             params['type'] = type
         if format is not None:
             params['format'] = format
-        if content == False:
+        if content is False:
```

Comment:
That's an example where this change _may_ be wrong. I'm going to pull an example out from my hat for educational purpose:

```python
In [12]: f == False
Out[12]: True

In [13]: class Falseish:
    ...:
    ...:     def __eq__(self, other):
    ...:         if other is False:
    ...:             return True
    ...:

In [14]: f = Falseish()

In [15]: f == False
Out[15]: True

In [16]: f is False
Out[16]: False
```

I believe in this case you are right, and that `==` might be wrong, it actually may even need to be completely removed as `if content` differs form `if content == False` differs from `if content is False`. Pulling an example out my hat again: 

```
In [17]: class Falseish:
    ...:
    ...:     def __bool__(self):
    ...:         return True
    ...:
    ...:     def __eq__(self, other):
    ...:         if other is False:
    ...:             return True
    ...:

In [18]: f = Falseish()

In [19]: if f: print('yeah')
yeah
```

typically empty string is falsish, and `if ''` behave as `if False`, wile `'' != False` and `''` is not `False` either. 
It's unclear (to me) without digging through history what the intent is, and the function is not documented, so that would be a good example of technical debt.